qutebrowser/utils/version.py

```diff
@@ -65,6 +66,7 @@ class DistributionInfo:
     pretty = attr.ib()
 
 
+PASTEBIN_URL = None
```

Comment:
ALL_CAPS is [reserved for constants](http://pep8.org/#constants), so this should be `pastebin_url`. I think it's okay to have a global here - using `objreg` for it wouldn't make it any less global (and I want to get rid of that anyways). Sometimes I wish Python had C-like static function variables :wink:

You could assign it to the function object (`pastebin_version.url = ...` and `getattr(pastebin_version, 'url', None)` but that seems a bit of a hack as well. I'll let you be the judge, I'm okay with both :wink:

Reply:
I think it's probably more readable to leave it as `pastebin_url =  None`, considering it's used for just one thing, but I'm willing to change it if you see fit